wcc-contentful/lib/wcc/contentful/store/cdn_adapter.rb

```diff
@@ -29,7 +29,7 @@ def initialize(client = nil, preview: false)
     end
 
     def find(key, hint: nil, **options)
-      options = { locale: '*' }.merge!(options || {})
+      options = options&.dup || {}
```

Comment:
Another case of our URLs changing.  Previously we would append `locale=*` to every query, now we don't.